### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by david-pepper**
## What does this change?

Create README.md to:

- Explain the functionality in the solution
- Add troubleshooting tips in case of alarm

### CODE REVIEW

**Comment by david-pepper** [comment]
[Emails to S3: Update Documentation](https://trello.com/c/YZv0ZnxL/2924-emails-to-s3-update-documentation)

**Review by kelvin-chappell — APPROVED** [review]
An outstanding amount of documentation! 💯

File: handlers/sf-emails-to-s3-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+#### Description
+The user credentials associated with the Lambda do not return a bearer token needed to query for records. 
+No records are retrievable without a bearer token. 
+#### Impact
+Emails will accrue in Salesforce.
+#### Potential Root Causes and Fixes
+##### Root Cause
+Running user is inactive or credentials expired
+##### Fix
+Inspect the credentials associated with the running user and verify that the user is active.
```
Comment (graham228221) [inline handlers/sf-emails-to-s3-exporter/README.md:57]:
```suggestion
### Metric: failed_to_authenticate_with_sf
#### Description
The user credentials associated with the Lambda do not return a bearer token needed to query for records. 
No records are retrievable without a bearer token. 
#### Impact
- Emails will accrue in Salesforce.
#### Potential Root Causes and Fixes
##### Root Cause
- Running user is inactive or credentials expired
##### Fix
- Inspect the credentials associated with the running user and verify that the user is active.
```
Formatting could be a little more readable here, but not sure the best way to do it. 

### Metric: failed_to_authenticate_with_sf
#### Description
The user credentials associated with the Lambda do not return a bearer token needed to query for records. 
No records are retrievable without a bearer token. 
#### Impact
- Emails will accrue in Salesforce.
#### Potential Root Causes and Fixes
##### Root Cause
- Running user is inactive or credentials expired
##### Fix
- Inspect the credentials associated with the running user and verify that the user is active.

File: handlers/sf-emails-to-s3-exporter/README.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+- failed_to_authenticate_with_sf
+- failed_to_get_records_from_sf
+- failed_s3_get_file
+- failed_s3_check_file_exists
+- failed_s3_write_file
+- failed_writeback_request_to_sf
+- failed_writeback_to_sf_record
+
+## Related Documentation
+[Shared Google Drive Folder: Sf Emails to S3](https://drive.google.com/drive/folders/1vSEIsatZ3DA3YQwGq0my8Gg5byDmTiOj?usp=sharing)
```
Comment (graham228221) [inline handlers/sf-emails-to-s3-exporter/README.md:43]: I'm going to create a label for this in our salesforce repo, perhaps we could link to that?

**Review by graham228221 — APPROVED** [review]
Looks great! and there's no "to-do" section haha

File: handlers/sf-emails-to-s3-exporter/README.md (hunk 2 above)
Comment (graham228221) [reply handlers/sf-emails-to-s3-exporter/README.md:43]: Label link: https://github.com/guardian/salesforce/pulls?q=label%3A%22Emails+to+S3%22

Comment (david-pepper) [reply handlers/sf-emails-to-s3-exporter/README.md:44]: Thanks! Added

File: handlers/sf-emails-to-s3-exporter/README.md (hunk 1 above)
Comment (david-pepper) [reply handlers/sf-emails-to-s3-exporter/README.md]: Added --- as a separator between trouble shooting items

File: handlers/sf-emails-to-s3-exporter/README.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        - Email Deletion when the emails are over 18 months old
+
+## Metrics
+- failed_to_authenticate_with_sf
+- failed_to_get_records_from_sf
+- failed_s3_get_file
+- failed_s3_check_file_exists
+- failed_s3_write_file
+- failed_writeback_request_to_sf
+- failed_writeback_to_sf_record
```
[the remark this answers is not in the document]
Comment (david-pepper) [reply handlers/sf-emails-to-s3-exporter/README.md]: I've had a bash at adding these as links, hopefully it works

File: handlers/sf-emails-to-s3-exporter/README.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
+# SF Emails to S3 Exporter
+
+## Description
+Performs 3 high level Operations:
+
+1. Exports emails from Salesforce to S3 (Lambda)
```
[the remark this answers is not in the document]
Comment (david-pepper) [reply handlers/sf-emails-to-s3-exporter/README.md]: Done

**Pull request merged**